packages/dialtone-icons/src/keywords.json

```diff
@@ -1358,72 +1358,20 @@
       ]
     },
     "general": {
-      "accessibility": [
-        "disability",
-        "disabled",
-        "dda",
-        "wheelchair"
-      ],
-      "add-task": [
-        "check",
-        "plus"
-      ],
-      "album": [
-        "photo",
-        "book"
-      ],
-      "archive-restore": [
-        "archive",
-        "unarchive",
-        "restore",
-        "index",
-        "box"
-      ],
-      "archive": [
-        "index",
-        "box"
-      ],
-      "attestation-outline": [
-        "checkbox"
-      ],
-      "attestation": [
-        "checkbox"
-      ],
-      "book-open": [
-        "read",
-        "library"
-      ],
-      "book": [
-        "read",
-        "dictionary",
-        "booklet",
-        "magazine",
-        "library"
-      ],
-      "bookmark-minus": [
-        "delete",
-        "remove"
-      ],
-      "bookmark-plus": [
-        "add"
-      ],
-      "bookmark": [
-        "read",
-        "clip",
-        "marker",
-        "tag"
-      ],
-      "box-select": [
-        "selection",
-        "square",
-        "rectangular",
-        "marquee",
-        "tool"
-      ],
-      "box": [
-        "cube",
-        "package"
-      ],
+      "accessibility": ["disability", "disabled", "dda", "wheelchair"],
+      "add-task": ["check", "plus"],
+      "album": ["photo", "book"],
+      "archive-restore": ["archive", "unarchive", "restore", "index", "box"],
+      "archive": ["index", "box"],
+      "attestation-outline": ["checkbox"],
+      "attestation": ["checkbox"],
+      "book-open": ["read", "library"],
+      "book": ["read", "dictionary", "booklet", "magazine", "library"],
+      "bookmark-minus": ["delete", "remove"],
+      "bookmark-plus": ["add"],
+      "bookmark": ["read", "clip", "marker", "tag"],
+      "box-select": ["selection", "square", "rectangular", "marquee", "tool"],
+      "box": ["cube", "package"],
```

Comment:
Is it your IDE that is doing this? This seems to keep happening every icon PR and then the linter automatically changes it when I go to do a release, lol.

Reply:
Haha, yeah, it looks like it was my IDE acting up. I've just reverted it. Thanks for catching that!